Dockerfile

```diff
@@ -1,16 +1,27 @@
-FROM node:6.12-slim
+FROM node:10.16-alpine
 
-ENV SCREENIE_VERSION=2.0.0
+ENV SCREENIE_VERSION=3.0.0-beta
 ENV SCREENIE_CHROMIUM_ARGS=--no-sandbox
+ENV SCREENIE_CHROMIUM_EXEC=/usr/lib/chromium/chrome
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
 
-RUN apt-get update && \
-    apt-get install -yq git ttf-ancient-fonts gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
-      libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
-      libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
-      libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \
-      ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget && \
-    wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64.deb && \
-    dpkg -i dumb-init_*.deb
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Installs make, g++, python and Chromium (77) packages.
```

Comment:
Can we align with Puppeteer 2.0.0 which comes with Chromium 79.0.3942.0 by default?